QMigrate/signal/scan.py

```diff
@@ -347,7 +351,9 @@ def _continuous_compute(self, starttime, endtime):
         for i in range(nsteps):
             w_beg = starttime + self.timestep*i - self.pre_pad
             w_end = starttime + self.timestep*(i + 1) + self.post_pad
-            logging.info("~"*20 + f" Processing : {w_beg}-{w_end} " + "~"*20)
+            logging.debug("~"*20 + f" Processing : {w_beg}-{w_end} " + "~"*20)
+            logging.info("~"*20 + f" Processing : {w_beg + self.pre_pad}-" + \
+                         f"{w_end - self.post_pad} " + "~"*20)
```

Comment:
```suggestion
            logging.info("~"*20 + f" Processing : {w_beg + self.pre_pad}-"
                         + f"{w_end - self.post_pad} " + "~"*20)
```

Python uses bracket closure to determine when to evaluate a statement, so don't need the `\`. It's also PEP8 convention to have binary operators come at the start of a new line when statements span more than one line.